src/main/webapp/js/components/interface/console/jsConsole.less

```diff
@@ -33,17 +33,21 @@
 }
 .consoleElement div.output {
 	display:block;
-	width:100%;
-	max-height:90%;
-	height: 87%;
-	position:relative;
+	width:99%;
+	max-height:80%;
+	height: 78%;
+	position:absolute;
 	overflow-y: auto;
 	overflow-x: hidden;
     padding-left: 2px;
     padding-right: 2px;
 	margin:0 0 10px;
 	border:0 none;
-	background:transparent
+	background:transparent;
+	top: 20px;
+	bottom: 50px;
+	right: 0px;
+	left: 2px;
```

Comment:
is this "left" here the one to touch to leave a bit more left margin on the console? maybe 5px?

Reply:
5px should be fine, I would not go more than that since there is also the output scroll bar on the right and if we push this too much that will disappear, I just tested it with 5 and seems nice.